operator/src/test/java/oracle/kubernetes/operator/helpers/VersionCheckTest.java

```diff
@@ -54,9 +54,9 @@ private static Stream<Arguments> getTestParams() {
           Arguments.of(LOG_MSG_TEST, "1", "12", "2", containsWarning(K8S_VERSION_TOO_LOW), noIgnores()),
           Arguments.of(LOG_MSG_TEST, "1", "14", "8", containsWarning(K8S_VERSION_TOO_LOW), noIgnores()),
           Arguments.of(LOG_MSG_TEST, "1", "15", "7", containsWarning(K8S_VERSION_TOO_LOW), noIgnores()),
-          Arguments.of(VERSION_TEST, "1", "16", "15", returnsVersion(1, 16), ignoring(K8S_VERSION_CHECK)),
-          Arguments.of(VERSION_TEST, "1", "17", "13", returnsVersion(1, 17), ignoring(K8S_VERSION_CHECK)),
-          Arguments.of(VERSION_TEST, "1", "18", "10", returnsVersion(1, 18), ignoring(K8S_VERSION_CHECK)),
```

Comment:
should we keep these lines for 1.16 - 1.18 but change them to expect containsWarning()? Should we also add lines for 1.19 and 1.20?

Reply:
Done